modules/core/src/main/scala/com/github/scoquelin/arugula/commands/internal/LettuceRedisBaseAsyncCommands.scala

```diff
@@ -0,0 +1,9 @@
+package com.github.scoquelin.arugula.commands.internal
```

Comment:
one other thought ... I could place all of these into the `com.github.scoquelin.arugula.commands` namespace to keep it symmetrical with how it is organized in the api layer.

Reply:
Yes I think that would be better to put everything in the `com.github.scoquelin.arugula.commands` except maybe for the lower level client implementations that should maybe remain in `client/internal`

Reply:
ok. that can be in a follow-on PR